gammapy/irf/rad_max.py

```diff
@@ -24,3 +31,51 @@ class RadMax2D(IRF):
 
     tag = "rad_max_2d"
     required_axes = ["energy", "offset"]
+
+
+    @classmethod
+    def from_irf(cls, irf: Union[EffectiveAreaTable2D, EnergyDispersion2D]):
```

Comment:
Note that so far we have not used typing in gammapy.

Reply:
Yes, will remove, this did just help my Editor with autocompletion